src/Rule/Order/OrderNoteAuthorRule.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Sdk\src\Rule\Order;
+
+use Exception;
+use MyParcelNL\Sdk\src\Rule\Rule;
+
+class OrderNoteAuthorRule extends Rule
+{
+    public const AUTHOR_CUSTOMER = 'customer';
+    public const AUTHOR_WEBSHOP  = 'webshop';
+    public const AUTHORS_ALLOWED = [
+        self::AUTHOR_CUSTOMER,
+        self::AUTHOR_WEBSHOP,
+    ];
+
+    /**
+     * @param $validationSubject
+     *
+     * @return void
+     * @throws \Exception
+     */
+    public function validate($validationSubject): void
```

Comment:
ik mis types

Reply:
`Rule` accepteert geen type? Ik heb het in de comment gezet.